core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -597,6 +600,13 @@ case class MisClassificationsPerCategory
   MisClassifications: Map[Double, Long]
 )
 
+case class labelPredictionConfidence
```

Comment:
@feifjiang are you planning to use LabelPredictionConfidenceCt and labelPredictionConfidence somewhere?

Reply:
@tovbinm  sorry - forgot to remove it. Just updated the PR again.